modules/drivers/googleanalytics/src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -30,10 +31,13 @@
     (:name (lib.metadata/field (qp.store/metadata-provider) id-or-name))
     id-or-name))
 
+(defn- truncate [t unit]
+  (u.date/truncate t unit {:first-day-of-week (public-settings/start-of-week)}))
+
```

Comment:
Duplicating this in a few places in the codebase doesn't seem ideal to me.

I think it's great to move the date helpers library out, but this feels like it still belongs in a `metabase.util.date` library. (Maybe bring that back when tweaking the aliases?)

Reply:
haha I did want to do that but changing all the aliases was my main reason not to. I think it's worth it tho because this code is just too noisy